migrations/0000_createDbs.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS "Users"(
+	id bigserial PRIMARY KEY,
+    "displayName" text NOT NULL,
+    "emailAddress" text UNIQUE NOT NULL,
+    password bytea NOT NULL,
+    "creationTime" timestamp NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "Publications"(
+	id bigserial PRIMARY KEY,
+    "authorId" bigint references "Users"(id) NOT NULL,
```

Comment:
why quotes around Users?

Reply:
omg is there some bullshit about postgres ignoring your case unless it's in quotes? if that's the case, i say fuck all capitalization in this file

Reply:
yes there is bullshit about caps without quotes. Want me to make all of the tables + fields lower case? so like authoerid?

Reply:
yeah looks like the convention is underscores so let's do `users` and `author_id`